juriscraper/opinions/united_states/federal_district/dcd.py

```diff
@@ -4,142 +4,102 @@
+        :param url:
+        :return:  docket document number and nature_of_suit
+        """
+        # In 2012 (and perhaps elsewhere) they have a few weird urls.
+        match = self.docket_document_number_regex.search(url)
+        if match:
+            doc_number = match.group(6)
+        else:
+            doc_number = url
 
-    def _get_docket_document_numbers(self):
-        document_numbers = []
-        regex = re.compile(r"(\?)(\d+)([a-z]+)(\d+)(-)(.*)")
-        for url in self.html.xpath(
-            "//table[2]//tr[position()>0]/td[3]/a/@href"
-        ):
-            # Because we are acting directly on the entire url list, no padding
-            # of the docket number field is required.
-            doc_no = regex.search(url)
-            # In 2012 (and perhaps elsewhere) they have a few weird urls.
-            if re.search(regex, url) is not None:
-                document_numbers.append(doc_no.group(6))
+        nature_of_suit_match = re.search(self.nature_of_suit_regex, url)
+        # In 2012 (and perhaps elsewhere) they have a few weird urls.
+        if not nature_of_suit_match:
+            nature_of_suit = "Unknown"
+        else:
+            nature_code = nature_of_suit_match.group(3)
+            if nature_code == "cv":
+                nature_of_suit = "Civil"
+            elif nature_code == "cr":
+                nature_of_suit = "Criminal"
+            # This is a tough call. Magistrate Cases are typically also
+            # Criminal or Civil cases, and their docket_number field will
+            # reflect this, but they do classify these separately under
+            # these 'mj' and 'mc' codes and the first page of these
+            #  documents will often refer to them as 'Magistrate Case
+            # ####-####' so, we will too.
+            elif nature_code == "mj" or "mc":
+                nature_of_suit = "Magistrate Case"
             else:
-                document_numbers.append(url)
-        return document_numbers
+                nature_of_suit = "Unknown"
 
-    def _get_judges(self):
-        judges = []
-        rowpath = "//table[2]//tr[position()>0]"
-        urlpath = "./td[3]/a/@href"
-        judgepath = "./td[3]"
-        for row in self.html.xpath(rowpath):
-            for judge_element in row.xpath(judgepath):
-                judge_string = html.tostring(
-                    judge_element, method="text", encoding="unicode"
-                )
-                judge = re.search(
-                    r"(by\s)(.*)", judge_string, re.MULTILINE
-                ).group(2)
-                # Determine the number of urls in each row and pad the judges
-                # list sufficiently
-                count = len(row.xpath(urlpath))
-                judges.extend([judge] * count)
-        return judges
+        return doc_number, nature_of_suit
+
+    def _get_docket_document_numbers(self):
+        return [case["docket_document_numbers"] for case in self.cases]
```

Comment:
Same here - is this necessary?

Reply:
They are. If removed, they will break the example tests

Reply:
which test?

Reply:
`tests.local.test_ScraperExampleTest`, since as is, each case would lose docket_document_numbers and nature_of_suit

https://github.com/freelawproject/juriscraper/blob/main/tests/examples/opinions/united_states/dcd_example.compare.json